test/integration/codeComponents/index.spec.ts

```diff
@@ -15,6 +15,13 @@ test.use({
   localAppConfig: {
     cmd: 'dev',
   },
+  ignoreConsoleErrors: [
+    // React setState warning inside "Recharts"
```

Comment:
Is there a bug in recharts? Or are we holding it wrong on toolpad side?

Reply:
I couldn't find a Recharts bug report, but also couldn't find anything we're doing with Recharts in the test that's failing

Reply:
Is it reproducible in isolation?

Reply:
Don't see any warnings in the console: https://codesandbox.io/p/sandbox/mutable-snow-x2kx55?file=%2Fsrc%2FApp.tsx

Reply:
As per DM, we should set `recharts` version from `alpha` to `^2.15.0`.